client/components/term-picker-option/style.scss

```diff
@@ -0,0 +1,77 @@
+.term-picker-option {
+  $padding-size: 18px;
+
+  display: flex;
+  width: 100%;
+  cursor: pointer;
+
+  padding: $padding-size;
+  border-radius: 5px;
+
+  background: #FFF;
```

Comment:
We should avoid using hex values for colors. We have a color variables define in `_colors.scss`, in this case you can use `$white`.

Reply:
Good call!

Reply:
@shaunandrews just replaced hex values with colors